src/plugins/offline_editing/offline_editing_plugin_gui.cpp

```diff
@@ -79,10 +78,10 @@ QVariant QgsSelectLayerTreeModel::data( const QModelIndex &index, int role ) con
           case Qt::ToolTipRole:
             return tr( "The source of this layer is a <b>WFS</b> server.<br>"
                        "Some WFS layers are not suitable for offline<br>"
-                       "editing due to unreliable/missing primary<br>"
-                       "keys, please check with your system<br>"
-                       "administrator if this WFS layer can<br>"
-                       "be used for offline editing." );
+                       "editing due to unstable primary keys<br>"
+                       "please check with your system administrator<br>"
+                       "administrator if this WFS layer can be used<br>"
```

Comment:
sorry, looks like it says "please check with you system administrator administrator if ..." now ;)